regions/shapes/line.py

```diff
@@ -0,0 +1,138 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from astropy.wcs.utils import pixel_to_skycoord, skycoord_to_pixel
+
+from ..core import PixCoord, PixelRegion, SkyRegion, BoundingBox
+
+__all__ = ['LinePixelRegion', 'LineSkyRegion']
+
+
+class LinePixelRegion(PixelRegion):
+    """
+    A line in pixel coordinates.
+
+    Parameters
+    ----------
+    start : `~regions.PixCoord`
+        Start position
+    end : `~regions.PixCoord`
+        End position
+    """
+
+    def __init__(self, start, end, meta=None, visual=None):
+        self.start = PixCoord._validate(start, name='start', expected='scalar')
```

Comment:
Should there be a better way to instantiate `PixCoord`s?  It seems weird to use private methods (`_validate`) here